addons/hr_timesheet/controllers/portal.py

```diff
@@ -48,7 +48,7 @@ def _get_searchbar_groupby(self):
         }
 
     def _get_search_domain(self, search_in, search):
-        search_domain = []
+        search_domain = FALSE_DOMAIN
```

Comment:
I don't know whether this is relevant, but if you don't go through any of the `if`s below, the original code was returning `[]` instead of `FALSE_DOMAIN`.  Maybe you need to reuse the pattern `OR(domains) if domains else []`.